composer/composerJSTutorial.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable func-style */
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module.exports = function main(
+  line = 'tampa, 106, january, null, null, 08--17-2019'
```

Comment:
looks like there's an extra hyphen in the date

Reply:
Thanks!